pkg/printer/alphabetical.go

```diff
@@ -0,0 +1,49 @@
+// alphabetical.go are the functions to get a sorted list of keys from sysls map types
+// Unfortunately go doesn't allow generics and pointer types don't implement empty interfaces
+// so map[string]interface{} doesn't cut it
```

Comment:
you could probably use `map[string]proto.Message` for this no?

Reply:
`Cannot use 'a.Attrs' (type map[string]*Attribute) as type map[string]*proto.Message`

because it's a map with the value being a pointer to a type the `proto.Message` thing doesn't work because a pointer